platform/osx/src/MGLMapView.mm

```diff
@@ -375,7 +375,6 @@ - (void)installAttributionView {
 
 /// Adds gesture recognizers for manipulating the viewport and selecting annotations.
 - (void)installGestureRecognizers {
-    self.acceptsTouchEvents = YES;
```

Comment:
I tested both of these before submitting, and both rotation and two-finger tap still work.